### PR TITLE
Updated manifest.json

### PR DESCRIPTION
**Pull request by octolexgaming**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/citymine/manifest.json [hunk 1]
```diff
@@ -12,16 +12,19 @@
     "citybuild": {
       "name": "Citybuild", 
       "color": "#ADD8E6",
+      "command": "/spawn",
```
Comment (DoJapHD) [inline minecraft_servers/citymine/manifest.json:15]: Command should join the gamemode not just go to spawn...

File: minecraft_servers/citymine/manifest.json [hunk 2]
```diff
@@ -12,16 +12,19 @@
...
       "versions": "1.9<1.21"
 	},
     "survival": {
       "name": "Survival", 
       "color": "#FFA500",
+      "command": "/spawn",
```
Comment (DoJapHD) [inline minecraft_servers/citymine/manifest.json:21]: Command should join the gamemode not just go to spawn...

File: minecraft_servers/citymine/manifest.json [hunk 3]
```diff
@@ -12,16 +12,19 @@
...
       "versions": "1.9<1.21"
 	},
     "grundstuecke": {
       "name": "Grundstücke", 
       "color": "#66FE32",
+      "command": "/spawn",
```
Comment (DoJapHD) [inline minecraft_servers/citymine/manifest.json:27]: Command should join the gamemode not just go to spawn...

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by octolexgaming** [comment]
> _Just as an information_: Your server join.citymine.eu **could be offline**. In general, we only accept pull requests from servers, **that are online**. Please change this, otherwise we cannot review your server correctly and have to deny the pull request.
> 
> If your server is currently online, then our api returned a wrong status, we will have a look at it :)
> 
> Reference: [API URL (https://api.mcsrvstat.us/2/join.citymine.eu)](https://api.mcsrvstat.us/2/join.citymine.eu)

Just a server restart it should be online again and I adjusted the manifest so the commands join the specific modes.

**Comment by DoJapHD** [comment]
Server is offline?

**Comment by octolexgaming** [comment]
> Server is offline?

It was just a server restart.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**